x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_investigate_in_timeline.tsx

```diff
@@ -163,6 +163,9 @@ export const useInvestigateInTimeline = ({
           columns: unifiedComponentsInTimelineEnabled ? defaultUdtHeaders : defaultHeaders,
           indexNames: timeline.indexNames ?? [],
           show: true,
+          excludedRowRendererIds: unifiedComponentsInTimelineEnabled
```

Comment:
For investigate in timeline I think we wanted the row renderers to be enabled, but I would confirm w/ product

Reply:
Sure. Let's talk to paul and for this PR, let keep one default and this small change can be made in BCs?

Reply:
👍🏾  sounds good!